options/posix/include/pthread.h

```diff
@@ -86,8 +86,10 @@ struct __mlibc_threadattr {
 	int __mlibc_inheritsched;
 	struct sched_param __mlibc_schedparam;
 	int __mlibc_schedpolicy;
+#if __MLIBC_LINUX_OPTION
```

Comment:
So, I don't really like this. For multiple reasons, one that this has to be an abi-break now which is kind of ugly. But it's also harmful that ABI now depends on compilation config. I'd much rather see this be included in the struct unconditionally, and that we have a definition for ``cpu_set_t`` internally (this can be in abi-bits maybe?). Then we can export ``cpu_set_t`` and the whole family of functions conditionally, but still have the ABI be constant.